docs/development/setup-vagrant.md

```diff
@@ -1091,6 +1092,38 @@ HOST_IP_ADDR 0.0.0.0
 (and restart the Vagrant guest), your host IP would be 0.0.0.0, a special value
 for the IP address that means any IP address can connect to your development server.
 
+### Customizing the virtual system settings
+
+When running Vagrant on top of Virtual Box or VM Ware Fusion, you can apply
+custom specifications for your VM.
```

Comment:
Maybe "When running Vagrant using a VM-based provider such as VirtualBox or VMWare Fusion, configuration determines what resources are allocated to the guest system."